provision/dev/nginx_site_docker.conf

```diff
@@ -4,7 +4,7 @@ server {
 
   location / {
   proxy_set_header X-Forwarded-Email "aaron.snow@gsa.gov";
-    proxy_set_header Host 192.168.33.10;
+    proxy_set_header Host 192.168.99.100:8000;
```

Comment:
Oh, good question--I did indeed get those values from the original Vagrant-specific [`provision/dev/nginx_site.conf`](https://github.com/18F/tock/blob/afd3ab719b09a7ba698e52a7126700534a2e21c3/provision/dev/nginx_site.conf).
One really annoying thing about nginx to me is that it's actually very difficult to even use environment variables in nginx configuration files.  I didn't realize that the specific `Host` value actually mattered--my impression was that it just needed to be some value considered "internal" so that the Django server would let it through, or something weird like that.

Does tock run on cloud.gov?  If nginx isn't absolutely critical for deployment, I'm wondering if we could write some simple WSGI middleware that injects these headers, instead of fronting the app with nginx during development.  I'm unfamiliar with the architecture of Tock, though, so I'm not sure if this would actually be a good solution...